modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -202,11 +215,44 @@ The Server Key is required to uniquely identify the server configuration. The re
 === Anypoint MQ Config
 
 [cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
-|======================
+|===
 | Field | Type | Description | Default Value | Required
 | Server Key | String | Identifier for the server configuration in AsyncAPI | None | Yes
 | Config Ref | String | Reference to the Anypoint MQ configuration details | None | No
-|======================
+|===
+
+[NOTE]
+The Server Key is required to uniquely identify the server configuration. Config Ref is optional and used to specify additional configuration details.
+
+[[SolaceConfig]]
+=== Solace Config
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|===
+| Field | Type | Description | Default Value | Required
+| Server Key | String | Server key set in the AsyncAPI spec | None | Yes
+| Config Ref | String | Reference to the configuration name of the Solace connection | None | No
+|===
+
+[NOTE]
+The Server Key is required to uniquely identify the server configuration. Config Ref is optional and used to specify additional configuration details.
+
+=== Limitations of Solace Connector
+
+The Solace connector in APIkit for AsyncAPI has the following limitations:
+
+* It doesn't support `topic listener`. It supports `queue listener`.
```

Comment:
Let me give some context, so that you can probably rephrase it better.
Each operation (message listener / publish) is bound to a `destination` - which can be a solace `queue` or `topic`. But Mulesoft Async API implementation supports only `queue` as a destination.

Reply:
@aby-philip-sf, let's see if my rewording works better.
Thanks!